### PR TITLE
FileManager: Plus dialog button from CoverBrowser

### PR DESCRIPTION
**Pull request by hius07**
The Plus dialog doesn't need an api, just fetch the button from the CoverBrowser.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/12857)

### CODE REVIEW

**Comment by NiLuJe** [comment]
Nice cleanup!

Friday night, so, take this with a grain of salt, but nothing jumps out ;).

File: frontend/apps/filemanager/filemanager.lua [hunk 1]
```diff
@@ -735,21 +736,27 @@ function FileManager:tapPlus()
...
                     callback = function()
-                        UIManager:close(self.file_dialog)
+                        UIManager:close(plus_dialog)
                         Device.importFile(self.file_chooser.path)
                     end,
                 },
             })
         end
+
+        local extract_button = self.coverbrowser and self.coverbrowser:genExtractBookInfoButton(close_dialog_callback)
```
Comment (poire-z) [inline frontend/apps/filemanager/filemanager.lua:746]: (Not fond of seeing named references to a plugin in core - but you have already done that a lot, so ok :))

**Review by poire-z — APPROVED** [review]

File: frontend/apps/filemanager/filemanager.lua (hunk 1 above)
Comment (hius07) [reply frontend/apps/filemanager/filemanager.lua:746]: I do hope one day you will agree it deserves to be moved to the core, it's the basis of the library.

Comment (hius07) [reply frontend/apps/filemanager/filemanager.lua:746]: Besides of the view, the database part is important. As we have measured, the sql request is faster than doc_settings opening.

Comment (Frenzie) [reply frontend/apps/filemanager/filemanager.lua:746]: I support someone other than me doing it. :innocent:

Comment (poire-z) [reply frontend/apps/filemanager/filemanager.lua:746]: > I do hope one day you will agree it deserves to be moved to the core

I have nothing against that (see https://github.com/koreader/koreader/issues/8472#issuecomment-1554638967) - as long as the database is still considered just a cache and trashable (and disable'able if possible).

> I support someone other than me doing it.

It's huge work, so: me too :)

Comment (Commodore64user) [reply frontend/apps/filemanager/filemanager.lua:746]: I think they are giving you the go ahead... @hius07

**Pull request merged**